modules/statistics/src/main/frontend/src/Statistics/Statistics.json

```
@@ -17,6 +17,8 @@
     "type": "reference",
     "primaryType": "cards:Question"
   },
+  "groupNullAndFalseXVar" : "boolean",
+  "groupNullAndFalseSplitVar" : "boolean",
```

The labels for these will be automatically generated as "Group null and false x var" and "Group null and false split var" which is not exactly human friendly. Here's a small improvement:
```suggestion
  "groupNullAndFalseAnswersForXVar" : "boolean",
  "groupNullAndFalseAnswersForSplitVar" : "boolean",
```

Additionally, since we now have 2 flags, the one ending in `xVar` one should be under the `xVar` field directly.